Cakefile

```diff
@@ -107,9 +107,9 @@ task "html", "Build HTML", (options) ->
   scripts = []
 
   if options.test
-    scripts.push "http://test.e17.dk/getnotaauthtoken"
+    scripts.push "http://nota.dk/getnotaauthtoken"
```

Comment:
Not at a pc right now, but if this is all the test setting does, then we should use the test servers here as you propose.

> Den 17. feb. 2016 kl. 12.10 skrev Daniel <notifications@github.com>:
> 
> In Cakefile:
> 
> > @@ -107,9 +107,9 @@ task "html", "Build HTML", (options) ->
> >    scripts = []
> >  
> >    if options.test
> > -    scripts.push "http://test.e17.dk/getnotaauthtoken"
> > +    scripts.push "http://nota.dk/getnotaauthtoken"
> Separate session and logins iirc. Server is also only available from the intranet.
> 
> —
> Reply to this email directly or view it on GitHub.
>